packages/@romefrontend/compiler/suppressions.ts

```diff
@@ -159,6 +158,33 @@ export function extractSuppressionsFromProgram(
 		}
 	}
 
+	// check for overlap suppressions
+	const nonOverlapSuppressions = new Map();
+	for (const suppression of suppressions) {
+		if (nonOverlapSuppressions.has(suppression["category"])) {
```

Comment:
For consistency

```suggestion
		if (nonOverlapSuppressions.has(suppression.category)) {
```

Thought about doing this automatically in the formatter but maybe we want to allow this usage to automatically escape certain lint rules? Not sure.